go/libraries/doltcore/schema/alterschema/addcolumn.go

```diff
@@ -37,18 +38,18 @@ const (
 // table, since we must write a value for each row. If the column is not nullable, a default value must be provided.
 //
 // Returns an error if the column added conflicts with the existing schema in tag or name.
-func AddColumnToTable(ctx context.Context, db *doltdb.DoltDB, tbl *doltdb.Table, tag uint64, newColName string, colKind types.NomsKind, nullable Nullable, defaultVal types.Value) (*doltdb.Table, error) {
+func AddColumnToTable(ctx context.Context, db *doltdb.DoltDB, tbl *doltdb.Table, tag uint64, newColName string, colNomsKind types.NomsKind, colDoltKind dolttypes.DoltKind, nullable Nullable, defaultVal types.Value) (*doltdb.Table, error) {
```

Comment:
All these double column kind params are an issue. There should be exactly one kind parameter for these methods, not two.